Code/Editor/EditorViewportWidget.h

```diff
@@ -65,130 +66,120 @@ namespace AzToolsFramework
 // EditorViewportWidget window
 AZ_PUSH_DISABLE_DLL_EXPORT_BASECLASS_WARNING
 AZ_PUSH_DISABLE_DLL_EXPORT_MEMBER_WARNING
-class SANDBOX_API EditorViewportWidget
+class SANDBOX_API EditorViewportWidget final
     : public QtViewport
-    , public IEditorNotifyListener
-    , public IUndoManagerListener
-    , public Camera::EditorCameraRequestBus::Handler
-    , public AzFramework::InputSystemCursorConstraintRequestBus::Handler
-    , public AzToolsFramework::ViewportInteraction::ViewportFreezeRequestBus::Handler
-    , public AzToolsFramework::ViewportInteraction::MainEditorViewportInteractionRequestBus::Handler
-    , public AzFramework::AssetCatalogEventBus::Handler
-    , public AZ::RPI::SceneNotificationBus::Handler
+    , private IEditorNotifyListener
+    , private IUndoManagerListener
+    , private Camera::EditorCameraRequestBus::Handler
+    , private Camera::CameraNotificationBus::Handler
+    , private AzFramework::InputSystemCursorConstraintRequestBus::Handler
+    , private AzToolsFramework::ViewportInteraction::ViewportFreezeRequestBus::Handler
+    , private AzToolsFramework::ViewportInteraction::MainEditorViewportInteractionRequestBus::Handler
```

Comment:
suggestion: It's fine to make notification buses private, but making request buses private isn't always a good idea, being able to call those functions through a pointer to `EditorViewportWidget` might be desired as the request bus forms par of the public API of `EditorViewportWidget` - I'd recommend leaving the request buses pubic and making the notification buses private (as they're purely an implementation detail of the type).